openerp/addons/base/ir/ir_ui_view.py

```diff
@@ -509,10 +509,15 @@ def read_combined(self, cr, uid, view_id, fields=None, context=None):
                     requested (similar to ``id``)
         """
         if context is None: context = {}
+        context = context.copy()
```

Comment:
Because i don't want to alter the context passed to this method.
I just want to add some values in context for methods called from here.